subprojects/execution/src/integTest/groovy/org/gradle/internal/execution/IncrementalExecutionIntegrationTest.groovy

```diff
@@ -804,6 +804,11 @@ class IncrementalExecutionIntegrationTest extends Specification {
                     throw new UnsupportedOperationException()
                 }
 
+                @Override
+                UnitOfWork.Incrementality getIncrementality() {
+                    return UnitOfWork.Incrementality.NOT_INCREMENTAL
```

Comment:
This is the `IncrementalExecutionIntegrationTest`, how come we are returning `NON_INCREMENTAL` here?

We need better names to distinguish between the two kinds of incrementality.

Reply:
Better names are welcome.